route-rs-runtime/src/element/dec_ip_hop.rs

```diff
@@ -0,0 +1,96 @@
+use crate::element::Element;
+use smoltcp::wire::*;
+
+/*
+  DecIpv4Hop
+  Decrements the TTL of an IPv4 packet
```

Comment:
Conventionally, we should use `///` comments here so they show in documentation. We also don't need to repeat the name in the header since it is shown immediately below.

Reply:
Seconded, we should begin migrating to docgen comments when it makes sense

Reply:
Done